src/random/isaac.cr

```diff
@@ -12,13 +12,10 @@ class Random::ISAAC
   private getter bb
   private getter cc
 
-  private alias Seeds = StaticArray(UInt32, 8)
-
   private def random_seeds
-    result = uninitialized Seeds
-    result_slice = result.unsafe_as(StaticArray(UInt8, sizeof(Seeds))).to_slice
-    Random::Secure.random_bytes(result_slice)
-    result
+    StaticArray(UInt32, 8).new do
```

Comment:
This is probably a bit slower than before (assuming the old code did it right) but this is on initialization and I prefer safe code even if a bit slower (not in hot path).